person.rb

```diff
@@ -10,9 +16,14 @@ def initialize(age = nil, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(book, date)
+    Rental.new(date, book, self)
   end
 
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- [optional] please kindly take your attr_accessor up before the beginning of your methods. As you know, it helps tell Ruby to create your methods for you. I.e. it creates both the READER & WRITER methods

Reply:
I've attended to that as well